core/src/main/resources/xsl/builder.xsl

```diff
@@ -53,22 +53,30 @@
     <xsl:param name="user" as="xsd:string" select="'unknown'" />
     <xsl:param name="creator" as="xsd:string" select="'unknown'"/>
     <xsl:param name="configMetadata" as="node()">
-        <meta>
-            <config option="checkXSDGrammar"  value="false"/>
-            <config option="checkJSONGrammar" value="false"/>
-            <config option="checkWellFormed"  value="false"/>
-            <config option="checkElements"    value="false"/>
-            <config option="checkPlainParams" value="false"/>
-            <config option="checkHeaders"     value="false"/>
+        <prams>
```

Comment:
That's what I meant, I can change this, but doesn't affect the outcome as long as meta is in a container.  Also note that configMetadata is *always* passed in, so this is more for debugging the XSLT outside of checker than anything else.